docs/api/structures/web-preferences.md

```diff
@@ -83,6 +83,10 @@
   window. Defaults to `false`. See the
   [offscreen rendering tutorial](../../tutorial/offscreen-rendering.md) for
   more details.
+* `offscreenUseSharedTexture` boolean (optional) _Experimental_ - Whether to use GPU shared texture for accelerated
```

Comment:
Since we haven't introduced new options for offscreen rendering prior to this PR, we're thinking of changing the `offscreen` property itself to accept an object or boolean.

* `offscreen` Object | Boolean (optional)
   * `useSharedTexture` boolean (optional) _Experimental_ - Whether to use GPU shared texture for accelerated color.

This would allow for future options to more easily be added.

We have precedence for this with the `titleBarOverlay` options if you're looking for a code reference.

https://github.com/electron/electron/blob/5b18cc46bc4ef293a5cfe54c52b94447bd7dc11c/shell/browser/native_window.cc#L111-L125

Reply:
Good idea!